### DISCUSSION TITLE
How Do I set the GUI's background to white to save the screenshot？

### DISCUSSION THREAD

**Discussion by huang-kaixin** [The-OpenROAD-Project/OpenROAD#6214] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/84197410-495a-441c-b328-7655a9079f19)
A white background like above instead of a black background below, Thank you for your reply!
![image](https://github.com/user-attachments/assets/bf10f85d-9664-4c08-8460-9db49f24cbea)

**Comment by maliberty** [comment]
The screen shots look like they are from different OS.  There is no background color option afaik (@gadfort confirm?)

**Reply by gadfort** [reply]
@maliberty that is correct, there is no way to change the background color

**Comment by mole99** [comment, marked as answer]
@huang-kaixin FYI this is now possible via #7214
